### PR TITLE
Migrate to new issue template

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
How it's render:
image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 - download the app
 - hop on your left foot three times
 - turn 90 degrees counterclockwise
 - head, shoulders knees and toes knees and toes knees and toes knees and toes knees and toes knees and toes knees and toes knees and toes knees and toes
+
+## Info
+
+- Device model: `Nexus 5`
+- Android version: `Android 14`
+- App version: `v1.3`
```
Comment (meenbeese) [inline .github/ISSUE_TEMPLATE.md:21]:
```suggestion
- App version: `v1.3.0`
```

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+## How to reproduce
 
 - download the app
 - hop on your left foot three times
 - turn 90 degrees counterclockwise
 - head, shoulders knees and toes knees and toes knees and toes knees and toes knees and toes knees and toes knees and toes knees and toes knees and toes
+
+## Info
+
+- Device model: `Nexus 5`
```
Comment (meenbeese) [inline .github/ISSUE_TEMPLATE.md:19]: Isn't Nexus 5 obsolete at this point? Can we instead use a newer Pixel device?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
 
-## Device Info
+---
 
-- Brand: `Samsunokiapplackberriaomissentialg`
-- Model name: `John Doe`
-- Android version: `Stock, hopefully.`
-
-## Steps to Reproduce
+## How to reproduce
```
Comment (meenbeese) [inline .github/ISSUE_TEMPLATE.md:10]:
```suggestion
## How to Reproduce
```

Comment (Jean-BaptisteC) [reply .github/ISSUE_TEMPLATE.md]: Done

**Comment by meenbeese** [comment]
Thanks!

**Pull request merged**